src/ocspdash/manager.py

```diff
@@ -524,11 +525,11 @@ def process_location(self, invite_token: bytes, public_key: str) -> Optional[Loc
 
         location = self.get_location_by_selector(selector)
         if location is None:
-            raise Exception(f'location not found for selector: {selector}')
+            raise ValueError(f'invalid invite token')
```

Comment:
Are these [vexing exceptions](https://ericlippert.com/2008/09/10/vexing-exceptions/)?

Reply:
No, since it doesn't mean your code has been designed wrong, but rather you have to deal with the reality that someone else is going to stick weird shit inside as the invite token.

Now, the next exception checking for the expiration of the pubkey might be different, but raising an exception actually makes the logic for using this function much better. Overall, I like this design and I think it does what it's supposed to in an elegant way. Lots of helpful/useful errors for bad situations where the program should stop seems good to me.